docs/03-apps-and-frameworks.md

```diff
@@ -0,0 +1,18 @@
+
+### Laravel
+  * [Account](http://account.stjude.cloud) / [Github](https://github.com/stjudecloud/account)
+  * [Viz Community](https://viz.stjude.cloud) / [Github](https://github.com/stjude/viz)
+  * [PeCan](https://pecan.stjude.cloud) / [Github](https://github.com/stjude/pecan) / `Uses Vue for Front-End`
+  * Vanguard / [Github](https://github.com/stjudecloud/vanguard) / `In Development`
+  * Intro to Genomics / [Github](https://github.com/stjude/introduction-to-genomics-for-engineers) / `In Development`
+
+### React
+  * [Platform](https://platform.stjude.cloud) / [Github](https://github.com/stjude/genomics-platform)
```

Comment:
I'd call it `Genomics Platform` since we are intentionally transitioning to that naming convention